webview/src/experiments/components/table/Indicators.tsx

```diff
@@ -115,19 +123,54 @@ const formatFilteredCountMessage = (filteredCounts: FilteredCounts): string =>
     .filter(Boolean)
     .join(', ')} Filtered`
 
+const addToSelected = (
+  selectedForPlotsCount: number,
+  row: Row<Experiment>
+): number => selectedForPlotsCount + (row.original?.selected ? 1 : 0)
+
+const getSelectedForPlotsCount = (rows: Row<Experiment>[] = []): number => {
+  let selectedForPlotsCount = 0
+
+  for (const row of rows) {
+    selectedForPlotsCount = addToSelected(selectedForPlotsCount, row)
+
+    selectedForPlotsCount =
+      selectedForPlotsCount + getSelectedForPlotsCount(row.subRows)
+  }
+
+  return selectedForPlotsCount
+}
+
 export const Indicators = ({
+  rows,
   sorts,
   filters,
   filteredCounts
 }: {
   sorts?: SortDefinition[]
   filters?: string[]
   filteredCounts: FilteredCounts
+  rows: Row<Experiment>[]
```

Comment:
[C] I don't think I should be giving this access to every single row. I will update to calculate the `selectedForPlotsCount` higher up.